cxotime/cxotime.py

```diff
@@ -486,7 +364,7 @@ class TimeSecs(TimeCxcSec):
     name = "secs"
 
 
-class TimeDate(TimeYearDayTime, FastDateParserMixin):
+class TimeDate(TimeYearDayTime):
```

Comment:
This makes sense and seems fine.  I note that I'm not immediately lear on how to initialize the astropy time classes, as the docstring doesn't seem to address the items in the init signature.  But that's not a cxotime issue.
```
Init signature:
TimeYearDayTime(
    val1,
    val2,
    scale,
    precision,
    in_subfmt,
    out_subfmt,
    from_jd=False,
)
Docstring:     
Year, day-of-year and time as "YYYY:DOY:HH:MM:SS.sss...".
The day-of-year (DOY) goes from 001 to 365 (366 in leap years).
For example, 2000:001:00:00:00.000 is midnight on January 1, 2000.

The allowed subformats are:

- 'date_hms': date + hours, mins, secs (and optional fractional secs)
- 'date_hm': date + hours, mins
- 'date': date
```

Reply:
These `TimeFormat` subclasses are defining a contained object *within* the `Time` class which is not directly set by the user.  The `Time` class is defined in `astropy/time/core.py`.